nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["src", "bin", "config", "lib", "routes"],
+    "ext": "js, tsx, ts, json",
+    "ignore": ["src/.next"],
+    "exec": "npm run dev"
```

Comment:
"npm run nodemon" will run "npm run dev" on reload. This is a good hot-reloader when working on the server